places-rest-apis/organizational-requirements.md

```diff
@@ -4,6 +4,6 @@ To access to the Places APIs, verify with your System administrator that the fol
 
 1. Places has been provisioned and appears in the organization. 
```

Comment:
Not sure whether this should stay "Places" or be changed to "Location Service", since it's in the "Web Service API" section. Will check and get back to you.